components/03-modules/gallery/mocks/gallery.json

```diff
@@ -0,0 +1,52 @@
+{
+  "class": "",
+  "typeHorizontal": "false",
```

Comment:
It's not a place to keep stuff like this, mocks should be used just to mimic the backend responses.